metpy/calc/thermo.py

```diff
@@ -22,6 +22,45 @@
 sat_pressure_0c = 6.112 * units.millibar
 
 
+@exporter.export
+@check_units('[temperature]', '[temperature]')
+def relative_humidity_from_dewpoint(temperature, dewpt):
+    r"""Calculate the relative humidity.
+
+    Uses temperature and dewpoint in celsius to calculate relative
+    humidity using the ratio of vapor pressure to saturation vapor pressures
+    and the Magnus Tetens equation
+
+    Parameters
+    ----------
+    temperature : `pint.Quantity`
+        The temperature
+    dew point : `pint.Quantity`
+        The dew point temperature
+
+    Returns
+    -------
+    `pint.Quantity`
+        The relative humidity
+
+    Notes
+    -----
+    Formula:
+    .. math :: e = \exp{(17.269*\frac{T_{d}-273.16}{T_{d}-35.86}})}
+    .. math :: e_{s} = \exp{(17.269*\frac{T-273.16}{T-35.86}})
+
+    .. math :: RH = \frac{e}{e_{s}} * 100
+
+    """
+    a = 17.269
+    e = 6.1078 * units.mb * np.exp(a * (dewpt - (273.16 * units.kelvin)) /
+                                   (dewpt - (35.86 * units.kelvin)))
+    e_s = 6.1078 * units.mb * np.exp(a * (temperature - 273.16 * units.kelvin) /
+                                     (temperature - 35.86 * units.kelvin))
+
+    return 100. * (e / e_s) * units.percent
```

Comment:
Can you just return `e / e_s`? We've been standardizing on having relative humidity as a dimensionless ratio (i.e. nominally in the range `[0, 1]`, see e.g. #677). Since you used units properly in the tests (great work!), you shouldn't even have to change the tests.